docs/tutorials/data_manipulation/dataframesdataset.md

```diff
@@ -0,0 +1,375 @@
+
+ds = DataFramesDataset(multiple_ts, target="target", freq=freq)
+train_and_predict(ds, prediction_length)
+```
+
+If the dataset is given as a long-dataframe, we can also use an
+alternative constructor `from_long_dataframe`. Note in this case
+we have to provide the `item_id` as well. Also note, if the
+`item_id`-column is called "item_id", we don't need to explicitly
+pass it to the constructor.
+
+
+```python
+N = 10
+ts_in_long_format = pd.concat(
+    [generate_single_ts(date_range, item_id=i) for i in range(N)]
+)
+
+# Note we need an item_id column now and provide its name (default is "item_id") to
+# the constructor. Otherwise, there is no way to distinguish different time series.
+ds = DataFramesDataset.from_long_dataframe(
+    ts_in_long_format, item_id="item_id", target="target", freq=freq
+)
+train_and_predict(ds, prediction_length)
+```
+
+## Include static and dynamic features
+
+The DataFramesDataset also allows us to include features of our time series.
+The list of all available features is given in the documentation.
+Here, we use dynamic real and static categorical features.
+To mimic those features we will just wrap our fancy data generator
+`generate_single_ts` and add a static cat and two dynamic real features.
+
+
+```python
+def generate_single_ts_with_features(date_range, item_id) -> pd.DataFrame:
+    ts = generate_single_ts(date_range, item_id)
+    T = ts.shape[0]
+    # static features are constant for each series
+    ts["static_cat_1"] = np.random.randint(low=0, high=2) # high is exclusive
+    ts["dynamic_real_1"] = np.random.normal(size=T)
+    ts["dynamic_real_2"] = np.random.normal(size=T)
+    # ... we can have as many static or dynamic features as we like
+    return ts
+
+ts = generate_single_ts_with_features(date_range, item_id=0)
+ts.head()
+```
+
+Now, when we create the gluonts dataset, we need to let the constructor
+know which columns are the categorical and real features. Also, we need a
+minor modification to the `train_and_predict` function. Here, we need to
+let the model know as well that more features are coming in. Note, if
+categorical features are provided, DeepAR also needs the cardinality as input.
+We have one static categorical feature that can take on two
+values (0 or 1). The cardinality in this case is a list of one element `[2,]`.
+
+
+```python
+def train_and_predict_with_features(ds, prediction_length):
+    model = DeepAREstimator(
+        freq=ds.freq,
+        prediction_length=prediction_length,
+        use_feat_dynamic_real=True,
+        use_feat_static_cat=True,
+        cardinality=[2,],
+        trainer=Trainer(epochs=1),
+    )
+    predictor = model.train(ds)
+    forecast_it, ts_it = make_evaluation_predictions(dataset=ds, predictor=predictor)
+    evaluator = Evaluator(quantiles=(np.arange(20) / 20.0)[1:])
+    agg_metrics, item_metrics = evaluator(ts_it, forecast_it, num_series=len(ds))
+    return agg_metrics["MSE"]
```

Comment:
This is very similar to `train_and_predict` above: should we have just one, taking as input the `Estimator` object, so it can be used all over the notebook?